src/lib/system-info.ts

```diff
@@ -1,17 +1,12 @@
 import * as systeminformation from 'systeminformation';
-import * as osUtils from 'os-utils';
 import * as _ from 'lodash';
 import { fs, child_process } from 'mz';
 
-export function getCpuUsage(): Promise<number> {
-	return new Promise((resolve) => {
-		osUtils.cpuUsage((percent) => {
-			resolve(Math.round(percent * 100));
-		});
-	});
+export async function getCpuUsage(): Promise<number> {
+	const cpuData = await systeminformation.currentLoad();
+	return Math.round(cpuData.avgLoad * 100);
```

Comment:
Wait I think I figured it out on https://github.com/sebhildebrandt/systeminformation/blob/master/lib/cpu.js#L1412-L1413, avgLoad is calculated as 

```
      let loads = os.loadavg().map(function (x) { return x / util.cores(); });
      let avgLoad = parseFloat((Math.max.apply(Math, loads)).toFixed(2));
``
[os.loadavg()](https://nodejs.org/api/os.html#os_os_loadavg) returns the 1,5 and 15 minute averages and the loads divides that into the number of cores and and avgLoad is the max of that calculation. I don't really understand the logic there. 

Maybe we can look at both values and see how the values compare to the output of top or htop?

Reply:
avgLoad is over time and currentLoad is on that cycle. The interval of each cycle is probably < 2 seconds.

When I chose avgLoad I did watch what the Supervisor was saying the cpu load was while monitoring top and they matches ~1% which I think was just based on round implementation. So top would say 19 and SV said 18.

Reply:
But if I understand correctly, if the device was under a high period of load, and then load drops, since `avgLoad` also considers the 15 minute load, then it would take some time to push the correct value to the cloud. Would you agree?